draft-tuexen-opsawg-pcapng.xml

```diff
@@ -403,8 +413,8 @@ Section Header
         </t>
         <t>[Open issue: does a program which re-writes a capture file change the original hardware/os/application info?]</t>
       </section>
-      <section anchor="sectionidb" title="Interface Description Block (mandatory)" toc="default">
-        <t>The Interface Description Block is mandatory. This block is needed to specify the characteristics of the network interface on which the capture has been made. In order to properly associate the captured data to the corresponding interface, the Interface Description Block MUST be defined before any other block that uses it; therefore, this block is usually placed immediately after the Section Header Block.  However, if capturing on an interface starts in the middle of a capture, an Interface Description Block can appear after other blocks, including blocks for packets on interfaces for which Interface Description Blocks have already appeared.</t>
+      <section anchor="sectionidb" title="Interface Description Block (optional)" toc="default">
+        <t>The Interface Description Block (IDB) is not mandatory, except in certain (common) cases: if the file contains an Enhanced Packet Block, Simple Packet Block, or Interface Statistics Block, then there MUST be an IDB in the file before those other blocks. This block is needed to specify the characteristics of the network interface on which the capture has been made. In order to properly associate the captured data to the corresponding interface, the Interface Description Block MUST be defined before any other block that uses it; therefore, this block is usually placed immediately after the Section Header Block.  However, if capturing on an interface starts in the middle of a capture, an Interface Description Block can appear after other blocks, including blocks for packets on interfaces for which other Interface Description Blocks have already appeared.</t>
```

Comment:
I'd just say "if the file contains a block that contains an interface ID, such as an Enhanced Packet Block, a Simple Packet Block, or an Interface Statistics Block, there must be an Interface Description Block for that interface before the first block that refers to that interface" - that states the requirement more generally.  If, for example, you have two EPBs, one referring to interface 0 and one referring to interface 1, it's not sufficient to have *any* IDB before the EPB referring to interface 1, you must have *two* IDBs, the first one being for interface 0 and the second one being for interface 1, before you have an EPB (or any other block) referring to interface 1.